.github/workflows/ci-external-scaler-azure-cosmos-db.yml

```diff
@@ -35,26 +39,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetesVersion: [v1.24, v1.23, v1.22, v1.21, v1.20, v1.19, v1.18, v1.17]
+        kubernetesVersion: [v1.24, v1.19]
```

Comment:
Should we test here the latest version only?

@tomkerkhove

Reply:
Let's follow our support matrix for KEDA and not x Kubernets versions. We should only test that, the rest is on their own risk imo

Reply:
So, can I merge this PR?

Reply:
Shouldn't we replace 1.24 with the latest one?

Reply:
I prefer to not touch their versions. I mean, if their scaler doesn't work on 1.28? Now I have just pruned up their versions and that's all, nothing more has changed